packages/v1-ready/zoho-crm/src/api.ts

```diff
@@ -48,6 +50,8 @@ export class Api extends OAuth2Requester {
             accounts: '/Accounts',
             account: (accountId: string) => `/Accounts/${accountId}`,
             accountSearch: '/Accounts/search',
+            webhooks: '/settings/automation/webhooks',
```

Comment:
See slack thread- wrong "webhooks". 

This is for Zoho Flow and I think their "automations" tooling inside CRM, i.e. as an end user I can configure a webhook to send data out to (Think: Webhooks by Zapier)

We're looking for "notifications" for the REST API in RM https://www.zoho.com/crm/developer/docs/api/v8/notifications/enable.html